Library/Homebrew/requirement.rb

```diff
@@ -31,7 +31,7 @@ def option_names
 
   # The message to show when the requirement is not met.
   def message
-    s = ""
+    s = "#{self.class} unsatisfied!\n"
```

Comment:
It's probably a non-issue (or an irrelevant corner case), as most in-formula requirements tend to provide their own `message` method, but if that wasn't the case, this would result in pretty ugly output like:

```console
$ brew install slimerjs
slimerjs: Formulary::FormulaNamespace4237eb49a894bb8c7e324716724ef7cb::FirefoxRequirement unsatisfied!

You can install with Homebrew Cask:
  brew cask install firefox
Error: An unsatisfied requirement failed this build.
```